tests/Configuration/test_Configuration.py

```diff
@@ -1,68 +1,78 @@
-from pytest import fixture, skip, mark
-from dabapush.Configuration.Registry import Registry
+"""Test suite for dabapush.Configuration.Configuration."""
+# pylint: disable=W0621
+from pytest import fixture, skip
+
 from dabapush.Configuration.ReaderConfiguration import ReaderConfiguration
+from dabapush.Configuration.Registry import Registry
 
 
 @fixture
 def conf():
+    """Provide a Registry."""
     return Registry()
 
 
 # should serialize
-def test_serialize(conf: Registry):
+def test_serialize():
+    """Should serialize the configuration"""
     skip()
 
 
 # should deserialize
-def test_deserialize(conf: Registry):
+def test_deserialize():
+    """Should deserialize the configuration"""
     skip()
 
 
 # should get a reader plugin and return it
 def test_get_reader(conf: Registry):
-    assert conf.get_reader("twacapic") is not None
+    """Should successfully get a ReaderConfiguration from the registry."""
+    assert conf.get_reader("Twacapic") is not None
 
 
 # should register a reader plugin and assign it a name
 def test_register_reader(conf: Registry):
+    """Should sucessfully register a new Class."""
+
     class Blub(ReaderConfiguration):
-        pass
+        """Yeah, some test class."""
 
     conf.register_reader("doopy", Blub)
 
     assert "doopy" in conf.readers
 
 
 # should remove reader plugin by name
-def test_remove_reader(conf: Registry):
+def test_remove_reader():
+    """Should do something, I guess."""
```

Comment:
Tests should be written at some point. Do we have an issue for that?

Reply:
Not yet.